frontend/app/pages/request_password_reset.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+import type { FormSubmitEvent } from '#ui/types'
+import { FetchError } from 'ofetch'
+import * as z from 'zod'
+
+definePageMeta({
+  middleware: 'guest',
+})
+
+type Schema = z.output<typeof schema>
+
+const schema = z.object({
+  email: z.email({ error: 'Adres e-mail jest wymagany' }),
+})
+
+const loading = ref(false)
+const toast = useToast()
+const state = reactive<Partial<Schema>>({
+  email: undefined,
+})
+
+const isButtonEnabled = computed(() => {
+  return schema.safeParse(state).success
+})
+
+async function onSubmit(event: FormSubmitEvent<Schema>) {
+  event.preventDefault()
+
+  loading.value = true
+
+  try {
+    toast.add({ title: 'Oczekiwanie', description: 'Wysyłanie emaila…', color: 'info' })
+
+    await useNuxtApp().$api('/auth/request_reset_password', {
+      method: 'POST',
+      credentials: 'include',
+      body: event.data,
+    })
+
+    toast.add({ title: 'Sukces', description: 'Pomyślnie wysłano link do resetowania hasła', color: 'success' })
+  } catch (error) {
+    console.error(error)
+    if (!(error instanceof FetchError)) {
+      throw error
+    }
+
+    if (error.data) {
+      toast.add({ title: 'Błąd logowania', description: error.data.message, color: 'error' })
+    } else {
+      toast.add({ title: 'Błąd logowania', color: 'error' })
+    }
+  } finally {
+    loading.value = false
+  }
+}
+</script>
+
+<template>
+  <AuthForm>
+    <h1 class="text-2xl font-medium">
+      Zresetuj hasło
+    </h1>
+
+    <UForm :schema="schema" :state="state" class="space-y-4 text-center" @submit="onSubmit">
```

Comment:
If email is invalid
![image](https://github.com/user-attachments/assets/cc5e7899-130d-4d20-a1ba-9e2a3ba37a77)